### ISSUE TITLE
colors

### ISSUE THREAD

**Issue by KashperIsCool** [billythegoat356/pystyle#14]
what colors can i use?

**Comment by TheBlackCookie** [comment]
if your just looking for colors I would use colorma.

**Comment by billythegoat356** [comment]
Yes, but pystyle offers a lot more colors than colorama. And also colorama can "break" the terminal.

**Comment by TheBlackCookie** [comment]
> Yes, but pystyle offers a lot more colors than colorama. And also colorama can "break" the terminal.

I agree but if your looking for the simple colors such as blue red green yellow black white(simple colors) then i would go for colorma 

If your looking for a bit more complex & customizable then I would use pystyle

**Comment by billythegoat356** [comment]
For simple colors yes you could use colorama 😁

**Comment by TheBlackCookie** [comment]
> For simple colors yes you could use colorama 😁

By the way mind showing me an example using the "error" function?

**Comment by billythegoat356** [comment]
Colorate.Error is pretty og
It will just use \n while coloring the text in red.
```python
print(Colorate.Error(" Error! This file does not exist!"))
```

**Comment by TheBlackCookie** [comment]
> Colorate.Error is pretty og
> It will just use \n while coloring the text in red.
> ```python
> print(Colorate.Error(" Error! This file does not exist!"))
> ```
> 

Mind attaching a screenshot of result? Not near my computer at the moment.

**Comment by billythegoat356** [comment]
i don't have my computer too 😅
but it just skips a line and color the text in red. really basic
